### PR TITLE
Add event and blog in community file

### PR DESCRIPTION
**Pull request by martindekov**
Adding talk in event tab in Lyon France by
Laurent Grangeau along with blog by Leon Sriger

Signed-off-by: Martin Dekov <mdekov@vmware.com>

Adding blog post and event

## Description

Adding blog post Artifactory and OpenFaaS – Containers Everywhere! by Leon Stigter and event in Lyon France Knative, OpenFaaS, Kubeless : la guerre des frameworks serverless est lancée ! by Laurent Grangeau

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))
- [x] Requested

## How Has This Been Tested?

N/A

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Community file change
## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ It would be great to hear from you especially if you have any of the above and w
 
 | Event name and description                                          | Speaker      | Location | Date        |
 |---------------------------------------------------------------------|--------------|----------|-------------|
+| Knative, OpenFaaS, Kubeless : la guerre des frameworks serverless est lancée ! | Laurent Grangeau | Lyon, France | 24-May-2019 |
```
Comment (alexellis) [inline community.md]: Looks like the URL is: https://mixitconf.org/

Comment (martindekov) [reply community.md]: Ok added it though the events for 2019 are not yet public

**Comment by alexellis** [comment]
Thank you

**Comment by martindekov** [comment]
My pleasure

**Pull request merged**